### PR TITLE
CPDRP-928: template updates

### PR DESCRIPTION
**Pull request by ethanmills**
## Ticket and context

Ticket: CPDRP-928

Kate asked for some changes to variables in notify templates

### Code quality checks
- [x] All commit messages are meaningful and true
- [x] Added enough automated tests

### CODE REVIEW

File: spec/features_helper.rb [hunk 1]
```diff
@@ -16,6 +16,8 @@
   Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
 end
 
+Capybara.server_port = 9887 + ENV["TEST_ENV_NUMBER"].to_i
+
```
Comment (kicferk1) [inline spec/features_helper.rb:20]: That's for parallel running of feature specs?

**Review by kicferk1 — APPROVED** [review]
I take it we plan to deploy this everywhere, add the new variables to notify templates, remove old variables from notify templates (if there are any), and clean up the code of them later?

File: spec/features_helper.rb [hunk 2]
```diff
@@ -16,6 +16,8 @@
   Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
 end
 
+Capybara.server_port = 9887 + ENV["TEST_ENV_NUMBER"].to_i
```
Comment (tonyheadford) [inline spec/features_helper.rb:19]: Is this for parallelising?

**Review by tonyheadford — APPROVED** [review]

File: spec/features_helper.rb (hunk 1 above)
Comment (ethanmills) [reply spec/features_helper.rb:20]: Yep, I was getting a intermittent failure where rspec was trying to start two servers on the same port

**Pull request merged**